sources/packages/web/src/components/students/applicationTracker/Completed.vue

```diff
@@ -98,6 +98,26 @@
     background-color="error-bg"
     content="You can review the outcomes of your requested changes in the table below by clicking “View request”. Please note your application will proceed without your requested changes, based on your last assessment."
   />
+  <application-status-tracker-banner
+    label="Attention! You are not yet eligible to receive funding."
+    icon="fa:fas fa-exclamation-circle"
+    icon-color="danger"
+    background-color="error-bg"
```

Comment:
The last AC in the ticket states that the card should be a waning "When warning card is displayed". Was there a decision to have it changed?

Reply:
I thought using the exclamation icon with danger for blocked disbursement tracker is more appropriate to draw attention from students. The screenshot below shows the warning tracker, which I think doesn't stand out as much.
![image](https://github.com/bcgov/SIMS/assets/148148914/1e9fedc7-1ae0-409f-bd10-bd703ff8aceb)

I can use warning tracker for the card according to AC.

Reply:
Independently of what we believe to be the best we should either follow the ACs or ask business for the confirmation. Either way is fine as far as we communicate and we are on the same page 😉

Reply:
Thanks for adjusting it 😉 
Should the `background-color` not be set as `warning-bg`?